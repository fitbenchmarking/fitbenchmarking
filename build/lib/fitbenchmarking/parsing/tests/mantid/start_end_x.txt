# Fitbenchmark Problem
software = 'Mantid'
name = 'start_end_x'
description = 'This dataset has no real world application. It is used to test the parsing.'
input_file = 'mantid_start_end_x.dat'
function ='name=UserFunction,Formula=x+100*a1+10*a2,a1=0,a2=1'
fit_ranges = {'x': [1.0, 3.0]}