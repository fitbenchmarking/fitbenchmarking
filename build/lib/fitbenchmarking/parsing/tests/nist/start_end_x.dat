NIST/ITL StRD
Dataset Name:  NA           (start_end_x.dat)

File Format:   ASCII
               Starting Values   (lines 29 to 29)
               Certified Values  (lines 31 to 34)
               Data              (lines 38 to 38)

Procedure:     Nonlinear Least Squares Regression

Description:   This test cannot be written in this data format.

Reference:     Fitbenchmark test.


Data:          1 Response Variable  (y = volume)
               0 Observations

Model:         Exponential Class
               1 Parameters (p1)

               y = p1 + e


 
        Starting values                  Certified Values
 
        Start 1       Parameter         Standard Deviation
  p1 =  0.0           N/A               N/A

Residual Sum of Squares:                    N/A
Residual Standard Deviation:                N/A
Degrees of Freedom:                                0
Number of Observations:                            0


Data:   y               x
        0               0
