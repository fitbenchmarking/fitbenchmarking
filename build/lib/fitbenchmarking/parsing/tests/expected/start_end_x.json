{"name": "start_end_x",
 "start_x": 1.0,
 "end_x": 3.0,
 "data_x": [0.5, 1.0, 1.5, 2.0, 2.5, 3.0, 3.5],
 "data_y": [33.5, 1.0, 0.1, -2.0, 0, -1.0, 6.2],
 "data_e": null,
 "function": "Single_function_2_params",
 "starting_values": [{"p1": 0.0, "p2": 1.0}],
 "value_ranges": null
}